released/packages/coq-coqeal/coq-coqeal.1.0.6/opam

```diff
@@ -0,0 +1,49 @@
+
+opam-version: "2.0"
+maintainer: "Cyril Cohen <cyril.cohen@inria.fr>"
+
+homepage: "https://github.com/CoqEAL/coqeal"
+dev-repo: "git+https://github.com/CoqEAL/coqeal.git"
+bug-reports: "https://github.com/CoqEAL/coqeal/issues"
+license: "MIT"
+
+synopsis: "CoqEAL - The Coq Effective Algebra Library"
+description: """
+This libary contains a subset of the work that was developed in the context of the ForMath european project (2009-2013). It has two parts:
+- theory (module CoqEAL_theory), which contains formal developments in algebra and optimized algorithms on mathcomp data structures.
+- refinements (module CoqEAL_refinements), which is a framework to ease change of data representation during a proof."""
+
+build: [make "-j%{jobs}%" ]
+install: [make "install"]
+depends: [
+  "coq" {>= "8.10" & < "8.15~"}
+  "coq-bignums" 
+  "coq-paramcoq" {(>= "1.1.1") | (= "dev")}
+  "coq-mathcomp-multinomials" {((>= "1.5.1" & < "1.7~") | = "dev")}
+  "coq-mathcomp-algebra" {((>= "1.12.0" & < "1.14~") | = "dev")}
+]
+
+tags: [
+  "category:Computer Science/Decision Procedures and Certified Algorithms/Correctness proofs of algorithms"
+  "keyword:effective algebra"
+  "keyword:elementary divisor rings"
+  "keyword:Smith normal form"
+  "keyword:mathematical components"
+  "keyword:Bareiss"
+  "keyword:Karatsuba multiplication"
+  "keyword:refinements"
+  "logpath:CoqEAL"
+]
+authors: [
+  "Guillaume Cano"
+  "Cyril Cohen"
+  "Maxime Dénès"
+  "Anders Mörtberg"
+  "Damien Rouhling"
+  "Vincent Siles"
+]
+
+url {
+  src: "https://github.com/CoqEAL/coqeal/archive/1.0.5.tar.gz"
+  checksum: "sha256=864a5e153ef27f1f608b0904966ae5fb9f6c00cc2945ab1016835e5339aed6b9"
```

Comment:
```suggestion
  src: "https://github.com/CoqEAL/coqeal/archive/1.0.6.tar.gz"
  checksum: "sha256=8632016a8a23678fe7a90c33206d90c40da3d85e77bf0bfb77da286d7e9711ee"
```
I assume you don't want the tarball and checksum for the *old* release.

Reply:
Indeed, thanks!